@here/harp-examples/src/getting-started_camera-animations.ts

```diff
@@ -0,0 +1,220 @@
+        orbit: false,
+        flyTo: "",
+        flyOver: false
+    };
+    let cameraAnimation: CameraKeyTrackAnimation | undefined;
+
+    // snippet:harp_gl_camera_animation_3.ts
+    const flyOverAnimationOptions = {
+        controlPoints: [
+            new ControlPoint({
+                geoCoordinate: geoLocations.BerlinReichstag,
+                timestamp: 0,
+                azimuth: 300,
+                altitude: 45,
+                distance: 500
+            }),
+            new ControlPoint({
+                geoCoordinate: geoLocations.BerlinStation,
+                timestamp: 10,
+                azimuth: 20,
+                altitude: 45,
+                distance: 3000
+            }),
+            new ControlPoint({
+                geoCoordinate: geoLocations.BerlinTower,
+                timestamp: 15,
+                azimuth: 180,
+                altitude: 35,
+                distance: 200
+            })
+        ]
+    };
+    // end:harp_gl_camera_camera_animation_3.ts
+
+    const gui = new GUI({ width: 300 });
+    gui.add(options, "globe").onChange(() => {
+        map.projection = options.globe ? sphereProjection : mercatorProjection;
+    });
+    gui.add(options, "orbit")
+        .onChange(enableOrbit)
+        .listen();
+    gui.add(options, "flyTo", ["", "BerlinTower", "BerlinReichstag", "Dubai", "BerlinStation"])
+        .onChange(flyTo)
+        .listen();
+    gui.add(options, "flyOver")
+        .onChange(enableFlyOver)
+        .listen();
+
+    function stopAnimation() {
+        if (cameraAnimation) {
+            cameraAnimation.stop();
+            cameraAnimation = undefined;
+            options.flyTo = "";
+            options.flyOver = false;
+            options.orbit = false;
+        }
+    }
+    function enableOrbit(enable: boolean) {
+        stopAnimation();
+        options.orbit = enable;
+        if (enable) {
+            cameraAnimation = createOrbitAnimation();
+            cameraAnimation.start();
+        }
+    }
+
+    function flyTo(location: string) {
+        stopAnimation();
+        options.flyTo = location;
+        if (location !== "") {
+            // snippet:harp_gl_camera_animation_1.ts
+            const target = new ControlPoint({
+                geoCoordinate: geoLocations[location] as GeoCoordinates,
+                distance: 800,
+                altitude: 25,
+                azimuth: Math.random() * 360,
+                timestamp: 10
+            });
+            const flyToOpts = CameraAnimationBuilder.createBowFlyToOptions(
+                map,
+                {
+                    ...CameraAnimationBuilder.getPoseFromView(map),
+                    timestamp: 0
+                },
+                target,
+                5000,
+                10
+            );
+            cameraAnimation = new CameraKeyTrackAnimation(map, flyToOpts);
+            // end:snippet:harp_gl_camera_animation_1.ts
+            cameraAnimation.start();
```

Comment:
I think this API is a bit complicated.
I would exect sth. like
```
MapView.lookAt({
    target,
    tilt,
    heading,
    zoomLevel,
    animation: MapView.BOW_ANIMATION
});
```
So there are two fundamental questions:
- Which module should contain the animations harp-mapview or harp-map-controls
- How does the API work. With a separate class (i.e CameraAnimation) or inside mapview

Reply:
Regarding the API, I think they do not exclude each other and an extra parameter in the lookAt function could live in parallel with the CameraAnimationBuilder,and therefore give the user the choice to use what is needed. Although I am a bit hesitant to add the whole Animation logic to the MapView, as in many use cases it might not be needed, or users might want to add their own modules for the animation. 
Already in the harp.gl examples there is 3 ways on how animate, in the webpage and test another 2, so we might not want to limit users to this one way of adding animations.

Regarding the module, I think this is interconnected with the API question, if the animations should be tightly interwoven in the MapView (like a parameter in the lookAt) and will become a dependency anyway, then they might be better placed in the harp-mapview module. If we keep them as a separate addon like the current MapAnimations and Controls, there would be the place.

Reply:
I like that this doesn't add anything to the MapView, it could be done in future, but I think best keep it as separate for as long as possible.